system/system_monitor/reader/traffic_reader/traffic_reader_service.cpp

```diff
@@ -153,6 +157,19 @@ void TrafficReaderService::start_nethogs(boost::archive::text_iarchive & archive
   thread_ = std::thread(&TrafficReaderService::execute_nethogs, this);
 }
 
+void TrafficReaderService::stop_nethogs()
+{
+  syslog(LOG_INFO, "Stopping nethogs...\n");
```

Comment:
A Same Point. `Skipping`

Reply:
Fixed in fad266d995c46e12f8ff763e4db67cb19a64298d